### ISSUE TITLE
Building v1.1.14 on macOS High Sierra with Macports

### ISSUE THREAD

**Issue by afraazali** [slact/nchan#441]
While building I seem to get the following error:

```
Undefined symbols for architecture x86_64:
  "_memrchr"
```

I've reverted back to v1.1.0 for the time being, which still works when building with Macports. I've also taken a look at the Homebrew formula, and the formula is currently using v1.1.0 as well.

**Comment by dcs3spp** [comment]
I also get the error when building from source on macOS HighSierra.

```
nginx: [emerg] dlopen() "/usr/local/nginx/modules/ngx_nchan_module.so" failed (dlopen(/usr/local/nginx/modules/ngx_nchan_module.so, 10): Symbol not found: _memrchr
  Referenced from: /usr/local/nginx/modules/ngx_nchan_module.so
  Expected in: flat namespace
 in /usr/local/nginx/modules/ngx_nchan_module.so) in /usr/local/nginx/conf/nginx.conf:5
```

Any ideas?

**Comment by slact** [comment]
Are you still having this issue with 1.1.15?

**Comment by afraazali** [comment]
I still have the same issue with 1.1.15.

**Comment by afraazali** [comment]
I can confirm that version 1.1.13 is the latest that I can compile nginx with using Macports. Looking at the brew formula, that is the current version its using as well.

Hopefully this helps. Thank you.

**Comment by slact** [comment]
fixed in e023c63
